domains/jeanrondon.json

```
@@ -7,6 +7,6 @@
         "twitter": "r4yb4"
     },
     "record": {
-        "CNAME": "jeandv.vercel.app"
+        "CNAME": "https://jeandv.github.io/my-next-portfolio/"
```

CNAME can't contain protocol or `/`
```suggestion
        "CNAME": "jeandv.github.io"
```